frontend/src/pages/protected/dashboard.tsx

```diff
@@ -1,6 +1,10 @@
-import React from "react"
+import React, {useEffect} from "react"
+import {Cookies} from "react-cookie"
 
 function Dashboard() {
+  const cookies = new Cookies()
+
+  useEffect(() => console.log(cookies.getAll(), []))
```

Comment:
remove `console.log()` we don;t want to log our code in the client side

Reply:
Pushed new commit for the fix.